components/organisms/InsightPage/DeleteInsightPageModal.tsx

```diff
@@ -0,0 +1,68 @@
+import Button from "components/atoms/Button/button";
+import TextInput from "components/atoms/TextInput/text-input";
+import Text from "components/atoms/Typography/text";
+import Title from "components/atoms/Typography/title";
+import { Dialog, DialogContent, DialogTitle } from "components/molecules/Dialog/dialog";
+import { FC, useState } from "react";
```

Comment:
nit: Third-party imports should be at the top for ease of visibility. Not a blocker

```suggestion
import { FC, useState } from "react";
import Button from "components/atoms/Button/button";
import TextInput from "components/atoms/TextInput/text-input";
import Text from "components/atoms/Typography/text";
import Title from "components/atoms/Typography/title";
import { Dialog, DialogContent, DialogTitle } from "components/molecules/Dialog/dialog";
```